releasetasks/templates/mobile/candidates_fennec.yml.tmpl

```diff
@@ -40,7 +40,7 @@
                 - /bin/bash
                 - -cx
                 - >
-                  cd /home/worker/checkouts/gecko && ln -s /home/worker/artifacts artifacts && ./mach --log-no-times taskgraph decision --pushlog-id='0' --pushdate='0' --project='{{ branch }}' --message='try: -b o -p foo -u none -t none' --owner='asasaki@mozilla.com' --level='3' --base-repository=$GECKO_BASE_REPOSITORY --head-repository=$GECKO_HEAD_REPOSITORY --head-ref=$GECKO_HEAD_REF --head-rev=$GECKO_HEAD_REV --triggered-by='nightly' --target-tasks-method='candidates_fennec'
+                  cd /home/worker/checkouts/gecko && ln -s /home/worker/artifacts artifacts && ./mach --log-no-times taskgraph decision --pushlog-id='0' --pushdate='0' --project='{{ branch }}' --message='try: -b o -p foo -u none -t none' --owner='asasaki@mozilla.com' --level='3' --base-repository=$GECKO_BASE_REPOSITORY --head-repository=$GECKO_HEAD_REPOSITORY --head-ref=$GECKO_HEAD_REF --head-rev=$GECKO_HEAD_REV {% if branch == "mozilla-beta" %} --triggered-by='nightly' {% endif %} --target-tasks-method='candidates_fennec'
```

Comment:
ERROR: Permission to lundjordan/releasetasks.git denied to MihaiTabara.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

Could not push to your repository. I'll merge this and follow-up with a change myself.